python/bin/watchman-make

```diff
@@ -146,6 +163,17 @@ Define the root of the project.  The default is to use the PWD.
 All patterns are considered to be relative to this root, and the build
 tool is executed with this location set as its PWD.
 """)
+parser.add_argument('-x', '--exclude', type=str, nargs='+',
+                    help="""
+Define a set of filename patterns which, if matching, will mark files as
+ignored from the next --target definition.
+
+The pattern syntax is wildmatch style; globbing with recursive matching via
+'**'. Exclude patterns have higher priority than match patterns.
+""")
+parser.add_argument('-U', '--sockname', type=str, default=None, help="""
```

Comment:
For testing, we normally just set `WATCHMAN_SOCK` in the environment; pywatchman will pick that up